components/base/InlineCTA.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styled from "@emotion/styled";
+import arrowIcon from "../../public/images/arrow-right-orange.svg";
+import { GREY_2, WHITE_PRIMARY, PARAGRAPH_TEXT } from "../colors";
+import { BOX_SHADOW_5 } from "../shadowDepths";
+
+const BoxStyle = styled.div`
+  height: 78px;
+  width: 656px;
+  box-sizing: border-box;
+  border: 1px solid ${GREY_2};
+  border-radius: 5px;
+  background-color: ${WHITE_PRIMARY};
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  :hover {
+    box-shadow: ${BOX_SHADOW_5};
+  }
+  @media only screen and (max-width: 656px) {
+    height: auto;
+    width: auto;
+  }
+`;
+const IconStyle = styled.div`
+  margin: 15px 33px 15px 15px;
+`;
+
+const TextStyle = styled.div`
+  height: 50px;
+  width: 506.84px;
```

Comment:
Oddly specific @ShreyaThapa, lol

Reply:
lol yeah, I was going off of the design mockups pixel to pixel, which I now understand does not have to be the case. haha